tests/page_test.go

```diff
@@ -31,6 +33,36 @@ type jsFrameBaseOpts struct {
 
 const sampleHTML = `<div><b>Test</b><ol><li><i>One</i></li></ol></div>`
 
+func TestNestedFrames(t *testing.T) {
+	t.Parallel()
+
+	tb := newTestBrowser(t, withFileServer(), withLogCache())
+	buf := bytes.NewBufferString("")
+	tb.logCache.entries[0].Logger.AddHook(&writer.Hook{Writer: buf, LogLevels: []logrus.Level{logrus.InfoLevel}})
+
+	page := tb.NewPage(nil)
+	_, err := page.Goto(tb.staticURL("iframe_test_main.html"), nil)
+	require.NoError(t, err)
```

Comment:
I think it would make things clearer if there were appropriate new line spaces between the different API calls, e.g. 

```go
	page := tb.NewPage(nil)
	_, err := page.Goto(tb.staticURL("iframe_test_main.html"), nil)
	require.NoError(t, err)

	frame1Handle, err := page.WaitForSelector("iframe[id='iframe1']", nil)
	assert.Nil(t, err)
	assert.NotNil(t, frame1Handle)
```

Reply:
done